### PR TITLE
TS-466: Integrating the top 10 trending companies

### PR DESCRIPTION
**Pull request by GadoliniumPotassium**
image
This PR is to introduce the integration of the top 10 trending companies to the trending companies tab. The tab now displays the company name and ticker as well as its price change throughout the day. At the current moment, it happens on page load.

### CODE REVIEW

**Comment by GadoliniumPotassium** [comment]
wtf is in this PR????

**Comment by GadoliniumPotassium** [comment]
there is shit in this branch that shouldnt be here if someone can help me remove them that would be great!

**Comment by tungleu** [comment]
image
is it possible if you can make those numbers green if it's positive and red if it's negative?

**Comment by GadoliniumPotassium** [comment]
> image is it possible if you can make those numbers green if it's positive and red if it's negative?

image
Change has been made

**Comment by AlyaNaseer** [comment]
If we don't have any functionality for the "3 dots", we should just remove it. What do you think? 
image

File: src/components/TrendingCompanies/CompanyCard.vue [hunk 1]
```diff
@@ -26,21 +38,42 @@
...
+  beforeMount () {
+    this.stockColorPicker()
+  },
+  methods: {
+    stockColorPicker () {
+      if (this.variation > 0) {
+        this.stockcolor = 'green'
+      } else {
+        this.stockcolor = 'red'
+      }
```
Comment (tungleu) [inline src/components/TrendingCompanies/CompanyCard.vue:76]: just a trick 😎 
```javascript 
this.variation > 0 ? this.stockcolor = 'green' : this.stockcolor = 'red'
```

File: src/hooks/useCredential.js [hunk 2]
```diff
@@ -34,3 +34,7 @@ export const logout = async (store, router) => {
 export const brokerageSignUp = async credentials => {
   return credentials
 }
+
+export const changePassword = async (credentials, store, router) => {
+  return null
+}
```
Comment (tungleu) [inline src/hooks/useCredential.js:40]: are these part of your PR?

File: src/components/TrendingCompanies/TrendingCompanies.vue [hunk 3]
```diff
@@ -36,45 +37,26 @@
...
+    }
+  },
+  beforeMount () {
+    this.pullCompanies()
+  },
+  methods: {
+    async pullCompanies () {
+      // If I am to change the fundamental design of trending companies then I will change the company card props
+      const response = await axiosInstace.get('/trendingCompanies')
+      console.log(response.data.trendingCompanies)
```
Comment (tungleu) [inline src/components/TrendingCompanies/TrendingCompanies.vue:58]: you can remove unnecessary comments and console.log here

**Review by tungleu — APPROVED** [review]

File: src/components/TrendingCompanies/CompanyCard.vue (hunk 1 above)
Comment (GadoliniumPotassium) [reply src/components/TrendingCompanies/CompanyCard.vue:76]: ok thank you

Comment (GadoliniumPotassium) [reply src/components/TrendingCompanies/CompanyCard.vue:76]: doing it this way is giving me a major code smell

**Review by tungleu — APPROVED** [review]

**Comment by GadoliniumPotassium** [comment]
> If we don't have any functionality for the "3 dots", we should just remove it. What do you think? image

do you want me to remove it?

**Comment by AlyaNaseer** [comment]
> > If we don't have any functionality for the "3 dots", we should just remove it. What do you think? image
> 
> do you want me to remove it?

Yeah I'd remove it

**Review by AlyaNaseer — APPROVED** [review]
Awesome work! LGTM

**Review by AlyaNaseer — DISMISSED** [review]
One quick thing I just noticed: you have the ticker under the company name just be plain text, but throughout our website, we include it in a blue label
image
image

Can you please change that? Should be a super quick change

Can you also center the contents:
image

**Comment by GadoliniumPotassium** [comment]
image
new look

**Review by AlyaNaseer — APPROVED** [review]
Awesome work!

**Review by tungleu — APPROVED** [review]

**Pull request merged**